bundles/org.openhab.binding.cloudrain/src/main/resources/OH-INF/binding/binding.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<binding:binding id="cloudrain" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:binding="https://openhab.org/schemas/binding/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/binding/v1.0.0 https://openhab.org/schemas/binding-1.0.0.xsd">
+
+	<name>Cloudrain Binding</name>
+	<description>This is the binding for the Cloudrain system.</description>
```

Comment:
Could you be a bit more verbose, so that somebody who hasn't heard about it gets a clue what this is about?

Reply:
This needs to be adapted to the new `addon.xml`, the PR to adapt the developer docs is not merged yet, so if you need more details, the corresponding issue is here: https://github.com/openhab/openhab-core/issues/2058
But you can also update your branch and look at another binding's `addon.xml` and adapt it to this initial contribution.